.circleci/sync-service.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
Nit, but this should be a deployment, not a daemon set - daemon set is something that is available on every single node of the cluster. I guess it is fine for tests, as we have only 1 node.

On the `infra` repo, this should be a deployment.